index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/0bd1156d30.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="Main-Page">
+     <nav class="nav">
+<div class="logo">AFIAMOAH</div>
+
+<div class="nav-links-menu"><p class="side-logo"><i class="fa-solid fa-bars"></i></p></div>
+
+     </nav>
+     <section class="section">
+<h1>Hey There. I’m Afiamoah</h1>
+<h2>I’am a Software Developer</h2>
+<p> class="content">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</p>
```

Comment:
- [ ] Please style your `p` tag correctly since it is not showing up in the browser as you can see below. It might have something to do with the text being black color by default.
![image](https://user-images.githubusercontent.com/90556221/176877804-c565cdfa-e845-4584-9509-2ef4bcae31c3.png)